### DISCUSSION TITLE
Use timestamp entities for highest and lowest

### DISCUSSION THREAD

**Discussion by Mariusthvdb** [JaccoR/hass-entso-e#45] [Ideas]
congrats on the first release. things are working as promised ;-) thanks for that!

suggestion: for the time sensors, please create an true timestamp entity, so we can set the format in Dashboards according to
https://www.home-assistant.io/dashboards/entities/#format

I am aware we can template it like this:

<img width="990" alt="Scherm­afbeelding 2022-10-28 om 23 40 09" src="https://user-images.githubusercontent.com/33354141/198737473-83718e07-beaa-4658-8fc0-b0fe6abc2506.png">

but that would obviously need yet another sensor....

asking because now it shows like this in the dashboard:

<img width="498" alt="Scherm­afbeelding 2022-10-28 om 23 41 19" src="https://user-images.githubusercontent.com/33354141/198737586-08f1af0b-c524-46cc-94c3-fc9e6b691973.png">

which is not very nice, with the +02:00

using a timestamp would fix that, and we could even do something like `format: relative `

thanks for considering!

**Comment by JaccoR** [comment]
This was the case some time ago, right? Somehow it changed. I'll have to look into it.

**Comment by JaccoR** [comment]
This seems to happen only when the name value is filled in. When it is not filled in it is working as expected. Weird...

**Comment by JaccoR** [comment]
It is in there in the new release.

**Reply by Mariusthvdb** [reply]
Updating now, will report back.

thanks!

**Reply by Mariusthvdb** [reply]
Perfect!

<img width="423" alt="Scherm­afbeelding 2022-10-29 om 20 10 01" src="https://user-images.githubusercontent.com/33354141/198846660-daf0df70-97d7-462a-ac94-60a7e7497547.png">

**Reply by Teme-V** [reply]
How you made that? for me it looks like in original post with +02.00 even using format: relative or not

**Reply by Mariusthvdb** [reply]
yes, the times areback  in the wrong format for a while now, I've got an issue on that matter somewhere...

here it is: https://github.com/JaccoR/hass-entso-e/issues/64
